### PR TITLE
Proper readme

### PR DESCRIPTION
**Pull request by rodrigodealer**
First version of our readme, just with the description.

@viniciushana

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Host detect
+
+Project that keeps scaning a list of configured  hosts and ports (or default ports) and exports the results to Prometheus.
```
Comment (alephmelo) [inline README.md]:
```suggestion
Project that keeps scaning a list of configured hosts and ports (or default ports) and exports the results to Prometheus.
```

Comment (rodrigodealer) [reply README.md]: Done. 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Host detect
+
+Project that keeps scaning a list of configured hosts and ports (or default ports) and exports the results to Prometheus.
```
Comment (viniciushana) [inline README.md]: :nail_care: sounds a bit too robotic. I'd rephrase it as `This program aims to...`

Comment (rodrigodealer) [reply README.md]: Done.

**Comment by alephmelo** [comment]
Any reason why we haven't merged this yet?
